app/views/phases/_submissions_table.html.erb

```diff
@@ -6,7 +10,16 @@
   <%= render 'shared/alert_error', alert_heading: t('alerts.recused_evaluator.heading'), alert_text: t('alerts.recused_evaluator.text') %>
 <% end %>
 
-<%= render partial: "phases/sort_and_filter" %>
+<div class="grid-container padding-x-0">
+  <div class="grid-row">
+    <div class="grid-col-6">
+      <%= render partial: "phases/sort_and_filter" %>
+    </div>
+    <div class="grid-col-6">
+      <%= render partial: "phases/export_submissions" %>
+    </div>
+  </div>
+</div>
```

Comment:
this seems fine but you could probably simplify the markup a little with a `flex-row flex-justify` here instead of using grid layout, to spread the sort_and_filter and export_submissions elements apart on the row.